jira_sync/repositories/pagure.py

```diff
@@ -86,10 +87,18 @@ def normalize_issue(self, api_result: dict[str, Any]) -> Issue:
             story_points=story_points,
         )
 
-    def get_issue_params(self) -> dict[str, Any]:
+    def get_issue_params(self, closed: bool) -> dict[str, Any]:
+        params = {}
+        if closed:
+            params = {
+                "status": "Closed",
+                "since": arrow.utcnow().shift(days=-self.retrieve_closed_days_ago).int_timestamp,
+            }
```

Comment:
This would become:

```
import datetime as dt
…
                "since": int(
                    (
                        dt.datetime.now(dt.UTC) - dt.timedelta(days=self.retrieve_closed_days_ago)
                    ).timestamp()
                ),
…
```